amis/packer_alinux.json

```diff
@@ -166,6 +167,12 @@
       "source" : "{{user `cookbook_path`}}",
       "destination" : "/tmp/cookbook"
     },
+    {
+      "type": "file",
+      "only": ["custom-alinux"],
+      "source": "{{user `script_path`}}",
+      "destination": "/tmp/script"
```

Comment:
let's be more specific in destination name, something like `/tmp/post_install_script_dir` or `/tmp/createami_scripts`